src/main/components/ViewTest.jsx

```diff
@@ -0,0 +1,258 @@
+      eyeColor: 'blue',
+      name: 'Trisha Dawson',
+      gender: 'female',
+      company: 'WAAB',
+      email: 'trishadawson@waab.com',
+      phone: '+1 (931) 434-2622',
+      address: '103 Lynch Street, Elrama, Montana, 3094',
+      about: 'Occaecat ullamco deserunt velit eu. Tempor exercitation veniam duis minim deserunt incididunt aute eu irure culpa consequat dolor aliquip. Minim quis aliquip do est do excepteur pariatur veniam. Nulla sunt ipsum quis eiusmod aliquip laborum. Mollit duis veniam qui aliquip. Nulla quis incididunt commodo eu eiusmod ipsum.\r\n',
+      registered: '2015-05-24T11:51:44 +04:00',
+      latitude: -26.547163,
+      longitude: 48.642857,
+      tags: [
+        'aliquip',
+        'commodo',
+        'consectetur',
+        'ullamco',
+        'et',
+        'eu',
+        'consectetur',
+      ],
+      friends: [
+        {
+          id: 0,
+          name: 'Cortez Bradley',
+        },
+        {
+          id: 1,
+          name: 'Dianne Michael',
+        },
+        {
+          id: 2,
+          name: 'Loretta Rodgers',
+        },
+      ],
+      greeting: 'Hello, Trisha Dawson! You have 2 unread messages.',
+      favoriteFruit: 'apple',
+    },
+    {
+      _id: '5cfc038aa88732edfb0cd254',
+      index: 4,
+      guid: '71dc7a55-64b1-4cd2-839f-60c18fabb998',
+      isActive: false,
+      balance: '$1,681.55',
+      picture: 'http://placehold.it/32x32',
+      age: 21,
+      eyeColor: 'brown',
+      name: 'Jacobs Barrera',
+      gender: 'male',
+      company: 'ENVIRE',
+      email: 'jacobsbarrera@envire.com',
+      phone: '+1 (873) 403-3402',
+      address: '735 Channel Avenue, Sutton, American Samoa, 2964',
+      about: 'Incididunt culpa quis dolor et ea eu officia excepteur labore eiusmod quis cillum cillum. Cupidatat eiusmod laboris aute sunt fugiat cupidatat aute dolor velit ipsum magna Lorem nostrud veniam. Eu id proident qui consequat qui incididunt. Magna officia voluptate ipsum non aute exercitation sint ad. Consectetur anim nulla eu amet tempor veniam ullamco ullamco consequat aliquip dolor aliqua adipisicing irure.\r\n',
+      registered: '2014-07-14T03:12:01 +04:00',
+      latitude: 59.477283,
+      longitude: -136.208247,
+      tags: [
+        'et',
+        'incididunt',
+        'minim',
+        'culpa',
+        'nulla',
+        'nostrud',
+        'aliquip',
+      ],
+      friends: [
+        {
+          id: 0,
+          name: 'Carver Moreno',
+        },
+        {
+          id: 1,
+          name: 'Figueroa Diaz',
+        },
+        {
+          id: 2,
+          name: 'Fran Park',
+        },
+      ],
+      greeting: 'Hello, Jacobs Barrera! You have 10 unread messages.',
+      favoriteFruit: 'banana',
+    },
+  ];
+  const renderTree = (div) => {
+    const tree = JsonTree.create(data, div);
+  };
+
+  useEffect(() => {
+    console.log('mounted');
+    const test = document.getElementById('tree');
+    console.log('test get : ', test);
```

Comment:
Is there a specific reason for these console logs?

Reply:
nope it was for testing, they are gone now